### PR TITLE
fixing gh action

### PR DESCRIPTION
**Pull request by hardikdava**

### CODE REVIEW

**Comment by hardikdava** [comment]
@AyushExel `epoch` is missing from cli argument. Otherwise training for 160 epochs will take longer in gh-action. Does it make sense to add it in cli?

**Comment by AyushExel** [comment]
For dfine_n_dummy the epoch should be hardcoded to 1 I think. 
Yes, there's plans to allow overriding almost all arguments from both cli and python, but it'll need some work as the dicts are nested. Lets do both separately. 
For the CI, is the dummy version with epoch 1 not being used?

**Comment by AyushExel** [comment]
I don't think model downloading is a problem, as predict tests are running fine locally.

**Comment by AyushExel** [comment]
Is the error related to model download code or just github env not allowing download? I can have a look in couple hours

**Comment by hardikdava** [comment]
@AyushExel I got tests working but taking total around 20 minutes. How can I train model with `224` image size instead of original image size?

**Comment by AyushExel** [comment]
I think you'll need to add size transform in the yml file for that. Maybe we can also create a smaller version of coco128 itself so 1 epoch takes much less.. I tried on roboflow but I think I'm out of credits..Let's merge this in for the time being..Slow tests are better than no tests

**Comment by AyushExel** [comment]
> @AyushExel Idk why but downloading models are from `{base_url}/{tag}/models/{model-name.pth}` here `models` were extra in model name. So `model_name`arg in `download_model` were used as `models/dfine-n.pth`

I think it should check both (-) & (_).. Its done to take user errors into account so for example if someone refers to model as dfine-n or dfine_n by mistake, it should automatically check for both.. Its weird that it works fine offline but fails in CI

File: trolo/utils/assets.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ def download_model(model_name: str, output_dir: str = ".") -> Optional[str]:
     output_dir = os.path.expanduser(output_dir)
     os.makedirs(output_dir, exist_ok=True)
 
+    model_name = model_name.replace("models/", "")
```
Comment (AyushExel) [inline trolo/utils/assets.py:38]: What is this for? We probably don't want to infer model name if user inputs a path like some/path/models/model.pth right?

File: trolo/utils/assets.py [hunk 2]
```diff
@@ -45,7 +47,7 @@ def download_model(model_name: str, output_dir: str = ".") -> Optional[str]:
     ]
 
     # GitHub release base URL
-    base_url = f"https://github.com/ayushexel/trolo/releases/download/{RELEASE_ASSETS_VER}"
+    base_url = f"https://github.com/AyushExel/trolo/releases/download/{RELEASE_ASSETS_VER}"
```
Comment (AyushExel) [inline trolo/utils/assets.py:50]: This one probably wont matter as username is not case sensitive

**Comment by hardikdava** [comment]
@AyushExel I have revert my changes of model paths. I can download coco128 from roboflow. Lmk.

**Comment by hardikdava** [comment]
@AyushExel Can you send me links to dataset? I can download it.

**Comment by AyushExel** [comment]
https://universe.roboflow.com/team-roboflow/coco-128/browse?queryText=&pageSize=50&startingIndex=0&browseQuery=true

Just select some 4-5 images and create a new dataset in coco format and zip it

**Comment by AyushExel** [comment]
Ok I think the problem is in this line, just change it from "models/dfine_n" to "dfine_n" https://github.com/AyushExel/trolo/blob/master/tests/test_det_infer.py#L161

**Comment by hardikdava** [comment]
@AyushExel here we go

[coco_subsample.v1i.coco.zip](https://github.com/user-attachments/files/17969272/coco_subsample.v1i.coco.zip)

**Comment by AyushExel** [comment]
Okay updated the asset.. Now the tests should take much less time.. Let me try to run it again

**Comment by hardikdava** [comment]
@AyushExel Yeah, it is much faster now.

**Pull request merged**